event_database/cloud_functions/external-data.go

```diff
@@ -107,6 +107,8 @@ func chainIdToCoinGeckoPlatform(chain vaa.ChainID) string {
 		return "klay-token"
 	case vaa.ChainIDCelo:
 		return "celo"
+	case vaa.ChainIDTerra2:
+		return "" // TODO
```

Comment:
This seems bad? Should this not return`terra2`?

Reply:
There isn't a CoinGecko asset_platform for terra2. When we add support for contract queries to get the contract's symbol, this shouldn't even need to be called for terra2